nondex-common/src/main/java/edu/illinois/nondex/shuffling/ControlNondeterminism.java

```diff
@@ -56,7 +56,7 @@ public static Configuration getConfiguration() {
     }
 
     public static <T> List<T> shuffle(List<T> originalOrder) {
-        return nondex.getPermutation(originalOrder);
+        return nondex == null ? originalOrder : nondex.getPermutation(originalOrder);
```

Comment:
If we remove this check when running mvn nondex:nondex we get a null pointed exception